src/server.py

```diff
@@ -146,16 +149,14 @@ def run_results_day_flot_query():
                  'osx10.6',
                  'osx10.7',
                  'osx10.8']
-    db = create_db_connnection()
 
     data_platforms = {}
     for platform in platforms:
-        cursor = db.cursor()
-        cursor.execute("""select DATE(date) as day,sum(result="%s") as failures,count(*) as
-                          totals from testjobs where platform="%s" and date >= "%s" and date <= "%s"
-                          group by day""" % ('testfailed', platform, start_date, end_date))
-
-        query_results = cursor.fetchall()
+        query_results = session.query(Testjobs.date.label('day'),
+                                      func.count(Testjobs.result == 'testfailed'
+                                                 ).label("failures"),
+                                      func.count(Testjobs).label('totals')).filter(
+            and_(Testjobs.date >= start_date, Testjobs.date <= end_date)).group_by('day').all()
```

Comment:
where is 'platform' in the query?  and are we sure the DATE(date) command in mysql maps to the same format as Testjobs.date.label('day')?